### PR TITLE
feat(icx-connect) : add past events tab

### PR DESCRIPTION
**Pull request by veanusnathan**
https://github.com/icx-id/icx-landing-page/assets/116778569/bd12c357-40b3-46ca-90fe-290a16387389

https://github.com/icx-id/icx-landing-page/assets/116778569/12b82768-3a29-4ede-b597-2d8989165662

https://github.com/icx-id/icx-landing-page/assets/116778569/c170049e-a7d2-49ba-b61a-d43f47c2d4f6

### CODE REVIEW

File: src/features/icxConnect/components/EventList.tsx [hunk 1]
```diff
@@ -8,13 +8,30 @@ import { EventScheduleType } from '../types';
 import { navigate } from 'gatsby';
 import { useMediaQuery } from '@mantine/hooks';
 
+const useStyles = createStyles(theme => ({
+  tab: {
+    fontWeight: 'bold',
+    color: 'gray',
+  },
+  selectInput: {
+    // borderRadius: '50px',
```
Comment (wibihanif) [inline src/features/icxConnect/components/EventList.tsx]: apaniccc

**Comment by ansorudin** [comment]
sekalian aja gasi desain in kalo event nya kosong

File: src/features/icxConnect/components/EventList.tsx [hunk 2]
```diff
@@ -8,13 +8,24 @@ import { EventScheduleType } from '../types';
...
 import { useMediaQuery } from '@mantine/hooks';
 
+const useStyles = createStyles(() => ({
+  tab: {
+    fontWeight: 'bold',
+    color: 'gray',
+  },
+}));
+
 function capitalizeFirstLetter(string: string) {
```
Comment (theodevoid) [inline src/features/icxConnect/components/EventList.tsx]: Bantu pisahin ini ke utils ya

File: src/features/icxConnect/components/EventList.tsx [hunk 3]
```diff
@@ -8,13 +8,24 @@ import { EventScheduleType } from '../types';
...
   return string.charAt(0).toUpperCase() + string.slice(1) || '';
 }
 
 export const EventList: React.FC<{}> = () => {
+  const { classes } = useStyles();
+
+  const [eventType, setEventType] = useState<EventScheduleType>(EventScheduleType.UPCOMING);
```
Comment (theodevoid) [inline src/features/icxConnect/components/EventList.tsx]:
```suggestion
  const [eventScheduleType, setEventScheduleType] = useState<EventScheduleType>(EventScheduleType.UPCOMING);
```

**Comment by theodevoid** [comment]
Tested for responsiveness?

**Review by theodevoid — DISMISSED** [review]
Minor changes, jangan lupa test responsiveness. Breakpoint tablet langsung ikutin style mobile aja

File: src/features/icxConnect/components/EventList.tsx (hunk 2 above)
Comment (veanusnathan) [reply src/features/icxConnect/components/EventList.tsx]: okay kak

**Comment by veanusnathan** [comment]
> Tested for responsiveness?

done ka the, check video recording nya yakk

**Review by theodevoid — APPROVED** [review]

**Pull request merged**